sigmastate/src/test/scala/special/sigma/SigmaDslSpec.scala

```diff
@@ -5,24 +5,27 @@ import java.math.BigInteger
 import org.ergoplatform.ErgoScriptPredef.TrueProp
 import org.ergoplatform.dsl.{SigmaContractSyntax, TestContractSpec}
```

Comment:
Unused import (and not only here in this file)

Reply:
done